pkg/core/native/management.go

```diff
@@ -17,19 +30,288 @@ type Management struct {
 	nextID  int32
 }
 
+// StoragePrice is the price to pay for 1 byte of storage.
+const StoragePrice = 100000
+
 const (
-	managementName        = "Neo Contract Management"
-	prefixContract        = 8
-	prefixNextAvailableId = 15
+	managementName = "Neo Contract Management"
```

Comment:
I think it is just `ManagementContract` https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/Native/NativeContract.cs#L29

Reply:
I see, it was changed in a later commit.